LIPS/lip-2.md

```diff
@@ -0,0 +1,248 @@
+---
+lip: 2
+title: Oracle contract upgrade to v2
+status: Work in progress
+author: Denis Glotov
+discussions-to: none
+created: 2021-02-22
+updated: 2021-02-24
+---
+
+# Oracle contract upgrade to v2
+
+The following changes are to be made to the first mainnet version of the oracle contract.
+
+
+## Change the meaning of 'quorum'.
```

Comment:
Changing the meaning of quorum and allowing removing oracles under quorum are two different changes, should separate

Reply:
Ok, I updated it, please have a look.